config/tutorials/otel-trace/index-config.yaml

```diff
@@ -5,51 +5,71 @@
 
 version: 0.6
```

Comment:
Is this index config used somewhere?

Reply:
Yes in tut https://github.com/quickwit-oss/quickwit/blob/main/docs/distributed-tracing/otel-service.md?plain=1#L30